### PR TITLE
Credits: Add release squad titles to credits section.

### PR DESCRIPTION
**Pull request by yogeshbhutkar**
Trac ticket: https://core.trac.wordpress.org/ticket/62386

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by apermo — APPROVED** [review]
Looking good to me.

File: src/wp-admin/credits.php [hunk 1]
```diff
@@ -135,3 +135,17 @@
 __( 'Release Deputy' );
 __( 'Core Developer' );
 __( 'External Libraries' );
+__( 'Release Coordinator' );
```
Comment (peterwilsoncc) [inline src/wp-admin/credits.php]: After a discussion on the thread, it was decided to go with `Release Coordination` as Coordinator is gendered in many languages that use gendered nouns, whereas coordination is not.

```suggestion
__( 'Release Coordination' );
```

**Review by peterwilsoncc — COMMENTED** [review]
This looks good, I've added a suggestion inline following the discussion on the thread.

File: src/wp-admin/credits.php (hunk 1 above)
Comment (yogeshbhutkar) [reply src/wp-admin/credits.php]: Thanks for the suggestion @peterwilsoncc. Implemented the changes in the latest commit 🚀.

**Review by peterwilsoncc — APPROVED** [review]
Thank you, looks good.

**Comment by peterwilsoncc** [comment]
Merged in [r59545](https://core.trac.wordpress.org/changeset/59545) / https://github.com/WordPress/wordpress-develop/commit/b8ecfbf9542198b1d88230221faeca6b71fa0ac7

Thanks for putting together the PR.

**Pull request closed without merging**